src/strategies/dynamic-whitelist/examples.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "name": "Example query",
+    "strategy": {
+      "name": "dynamic-whitelist",
+      "params": {
+        "symbol": "dynamic"
+      }
+    },
+    "network": "4",
+    "options": {
+      "contractAddress": "0x256669f7a136a99baf979deea862b38d9866d08e",
+      "etherscanApiKey": "95WX4VDE93FSS5SUR8QZ6VUWBF7QBTJIVE"
+    },
```

Comment:
These options should be passed from strategy.params, please refer to other strategies